docs/production.md

```diff
@@ -89,8 +90,10 @@ After installing a Katello server, you could then spin up a Capsule with the ass
 
 For a release version in production:
 
-    ansible-playbook -l localhost playbooks/katello.yml -e foreman_repositories_version=1.14 -e katello_repositories_version=3.3
+    ansible-playbook --connection=local --inventory=localhost, -l localhost playbooks/katello.yml -e foreman_repositories_version=1.14 -e katello_repositories_version=3.3
 
 For nightly production:
 
-    ansible-playbook -l localhost playbooks/katello.yml
+    ansible-playbook --connection=local --inventory=localhost, -l localhost playbooks/katello.yml
```

Comment:
I'd like to simplify this to having an inventory in `inventories/` that looks like:

```
localhost ansible_connection=local
```

Reply:
we can do that, yeah.